### PR TITLE
Update Client API details

### PR DESCRIPTION
**Pull request by OjesviPandita**
Updated Client API section till Navigation module.

### CODE REVIEW

File: docs/clients.md [hunk 1]
```diff
@@ -1,83 +1,70 @@
-# Ultra Client Side SDK
+# Ultra Client Side SDK documentation
+Here are the details of the Client side APIs you need to incorporate while integration of your app with Ultra.
 
-> Latest JS SDK version : [1.0.0-beta](https://www.npmjs.com/package/fk-platform-sdk)
+> Use this latest version of JS SDK: [1.0.0-beta](https://www.npmjs.com/package/fk-platform-sdk)
```
Comment (MSReehal) [inline docs/clients.md]: Please change "1.0.0-beta" to "1.0.5"

File: docs/clients.md [hunk 2]
```diff
@@ -1,83 +1,70 @@
...
-This SDK enables developers to build applications that run inside Flipkart app.
-
-All the methods mentioned here will work with both React Native and Webview. 
-All the methods are asynchronous in nature and will always return a promise that gets resolved with the values. Fire and forget calls are an exception where you may not care about the response.
-
-## Getting Started
+## Integrate JS SDK
 #### Add the dependency
-If using node, add this repository as an npm package (Both webview and react-native)
+If you are using a node, add the following repository as an NPM package for both WebView and React Native platforms. Visit [here](https://www.npmjs.com/package/fk-platform-sdk) to get the latest SDK for this repository.
```
Comment (MSReehal) [inline docs/clients.md]: Replace "a node" with "Node.js"

File: docs/clients.md [hunk 3]
```diff
@@ -1,83 +1,70 @@
...
 
 ```
 npm install --save fk-platform-sdk
 ```
 
-You can also visit [the NPM](https://www.npmjs.com/package/fk-platform-sdk) page for the SDK 
-
-Alternatively, if you are using webview only you can also include the following script directly inside a `<script>` tag:
+Or if you are using only a web browser without node, include the following script directly inside the `<script>` tag:
```
Comment (MSReehal) [inline docs/clients.md]: Please replace "only a web browser without node" with "webview only"

File: docs/clients.md (hunk 1 above)
Comment (OjesviPandita) [reply docs/clients.md]: Done, please check the latest commit.

File: docs/clients.md (hunk 2 above)
Comment (OjesviPandita) [reply docs/clients.md]: Done, please check the latest commit.

File: docs/clients.md (hunk 3 above)
Comment (OjesviPandita) [reply docs/clients.md]: Done, please check the latest commit.

File: docs/clients.md [hunk 4]
```diff
@@ -1,83 +1,70 @@
...
 
 ```
 https://img1a.flixcart.com/linchpin-web/fk-platform-sdk/fkext-browser-min@1.0.0-beta.1.js
```
Comment (OjesviPandita) [inline docs/clients.md]: Is there any change any change in "1.0.0-beta.1.js" here?

**Pull request merged**